libbeat/common/field_test.go

```diff
@@ -27,6 +27,58 @@ import (
 	"github.com/elastic/go-ucfg/yaml"
 )
 
+func TestFieldsHasNode(t *testing.T) {
```

Comment:
is this actually supposed to test `CanConcat`?

Reply:
Yes, will change.